docs/interpreter/cassandra.md

```diff
@@ -471,253 +548,285 @@ It is also possible to use dynamic forms for **prepared statements**:
+     <td>100</td>
+   </tr>
+   <tr>
+     <td>cassandra.pooling.new.connection.threshold.remote</td>
+     <td>Protocol V2 and below default = 100. Protocol V3 and above default = 200</td>
+     <td>100</td>
+   </tr>
+   <tr>
+     <td>cassandra.pooling.pool.timeout.millisecs</td>
+     <td>Cassandra pool time out in millisecs</td>
+     <td>5000</td>
+   </tr>
+   <tr>
+     <td>cassandra.protocol.version</td>
+     <td>Cassandra binary protocol version</td>
+     <td>3</td>
+   </tr>
+   <tr>
+     <td>cassandra.query.default.consistency</td>
+     <td>
+      Cassandra query default consistency level
+      <br/>
+      Available values: ONE, TWO, THREE, QUORUM, LOCAL_ONE, LOCAL_QUORUM, EACH_QUORUM, ALL
+     </td>
+     <td>ONE</td>
+   </tr>
+   <tr>
+     <td>cassandra.query.default.fetchSize</td>
+     <td>Cassandra query default fetch size</td>
+     <td>5000</td>
+   </tr>
+   <tr>
+     <td>cassandra.query.default.serial.consistency</td>
+     <td>
+      Cassandra query default serial consistency level
+      <br/>
+      Available values: SERIAL, LOCAL_SERIAL
+     </td>
+     <td>SERIAL</td>
+   </tr>
+   <tr>
+     <td>cassandra.reconnection.policy</td>
+     <td>
+        Cassandra Reconnection Policy.
+        Default = new ExponentialReconnectionPolicy(1000, 10 * 60 * 1000)
+        To Specify your own policy, provide the <strong>fully qualify class name (FQCN)</strong> of your policy.
+        At runtime the interpreter will instantiate the policy using 
+        <strong>Class.forName(FQCN)</strong>
+     </td>
+     <td>DEFAULT</td>
+   </tr>
+   <tr>
+     <td>cassandra.retry.policy</td>
+     <td>
+        Cassandra Retry Policy.
+        Default = DefaultRetryPolicy.INSTANCE
+        To Specify your own policy, provide the <strong>fully qualify class name (FQCN)</strong> of your policy.
+        At runtime the interpreter will instantiate the policy using 
+        <strong>Class.forName(FQCN)</strong>
+     </td>
+     <td>DEFAULT</td>
+   </tr>
+   <tr>
+     <td>cassandra.socket.connection.timeout.millisecs</td>
+     <td>Cassandra socket default connection timeout in millisecs</td>
+     <td>500</td>
+   </tr>
+   <tr>
+     <td>cassandra.socket.read.timeout.millisecs</td>
+     <td>Cassandra socket read timeout in millisecs</td>
+     <td>12000</td>
+   </tr>
+   <tr>
+     <td>cassandra.socket.tcp.no_delay</td>
+     <td>Cassandra socket TCP no delay</td>
+     <td>true</td>
+   </tr>
+   <tr>
+     <td>cassandra.speculative.execution.policy</td>
+     <td>
+        Cassandra Speculative Execution Policy.
+        Default = NoSpeculativeExecutionPolicy.INSTANCE
+        To Specify your own policy, provide the <strong>fully qualify class name (FQCN)</strong> of your policy.
+        At runtime the interpreter will instantiate the policy using 
+        <strong>Class.forName(FQCN)</strong>
+     </td>
+     <td>DEFAULT</td>
+   </tr>
+ </table>
+
+<hr/>
+
+## 14. Change Log
+
+**2.0** _(Zeppelin 0.5.7-incubating)_ : 
```

Comment:
I'm not sure we're planning to have 0.5.7 before we go 0.6.0.

If you mean current version of Zeppelin, i think you can define a variable `ZEPPELIN_VERSION` `https://github.com/apache/incubator-zeppelin/blob/master/docs/_config.yml`, like `BASE_PATH`.

And reference it in the md
```Zeppelin {{ ZEPPELIN_VERSION}} : ```